### PR TITLE
fix tests and compilation with `TAGS=libhamlib`

### PR DESCRIPTION
**Pull request by as3ii**
**Note: this requires the merge of https://github.com/la5nta/wl2k-go/pull/85**

### CODE REVIEW

**Comment by as3ii** [comment]
I don't know precisely if this is related to the use of libhamlib, and if this is an issue in pat or in wl2k-go, but sometimes randomly pat starts eating a lot of cpu power. The only way I was able to reproduce consistently this bug was manually disconnecting and reconnecting the usb serial interface to the radio.
This is what `perf` recorded and found when this happens:
perf

Tell me if I can do anything to help debug this problem

File: riglist.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ package main
 import (
 	"fmt"
 	"strings"
-
+	"context"
```
Comment (martinhpedersen) [inline riglist.go:13]: I believe we're missing a white space here. Did you [go fmt](https://go.dev/blog/gofmt) the code?

**Comment by martinhpedersen** [comment]
> I don't know precisely if this is related to the use of libhamlib, and if this is an issue in pat or in wl2k-go, but sometimes randomly pat starts eating a lot of cpu power.

Thanks! I don't think i have seen this when Pat is compiled without `TAGS=libhamlib`. The feature is rarely used and not enabled in the official releases, so I don't think I'll prioritize debugging this any time soon. Feel free to debug further if you feel like it 🙂 

PS: Thanks for the PR! ❤️

File: riglist.go (hunk 1 above)
Comment (as3ii) [reply riglist.go:11]: added a commit after `go fmt`. It just reordered the entries

**Comment by martinhpedersen** [comment]
I've merged the wl2k-go changes and pulled them into Pat's develop branch. Please rebase this PR on top of `develop`. Thank you 🙂

**Comment by martinhpedersen** [comment]
Ooops! I did something wrong while trying to rebase this, and since the PR was automatically closed I no longer have permission to push to your branch.

I'll merge this manually to develop.

Thanks for the PR 😄

**Comment by martinhpedersen** [comment]
Merged (b62dbc787b294e63e30da568ac3ea423f2abbc97).

**Pull request closed without merging**